README.md

```diff
@@ -220,9 +220,9 @@ jobs:
       with:
```

Comment:
Can you add `- uses: actions/checkout@v3` as the first step in this example please. It seems to be lacking when I tried this code snippet